typedjsonrpc/registry.py

```diff
@@ -124,6 +134,7 @@ def _handle_exceptions(self, method, is_notification=False, msg_id=None):
                     debug_url = self._store_traceback()
                 else:
                     debug_url = None
+                self._logger.exception(exc_info[2])
```

Comment:
This looks like it would log the message, but it gives you no idea what input caused it? Could we do something more around returning the message with the id that it was sent out with? Or maybe we can store a combination of the name of the method and the message id? A debug url (if one exists) may be helpful too.